inscriptions/ISic003801.xml

```diff
@@ -189,7 +198,7 @@
         <body>
             <div type="edition" xml:space="preserve" xml:lang="grc" resp="#AA">
                 <ab>
-                    <lb n="1"/>Ἀ<supplied reason="lost">πο</supplied><unclear>λλ</unclear><supplied reason="lost">ω</supplied><unclear>ν</unclear>ίου
+                    <lb n="1"/><persName type="attested"><name><seg part="I">Ἀ</seg><supplied reason="lost">πο</supplied><seg part="M"><unclear>λλ</unclear></seg><supplied reason="lost">ω</supplied><seg part="F"><unclear>ν</unclear>ίου</seg></name></persName>
```

Comment:
in general, I think this is more than is necessary. We generally just use seg for those cases where we only have a single bit of a name that we still think is a name - if it's restoreable and/or broken into multiple bits, seg is unnecessary, since we can already distinguish names that include 'supplied' from the coding - seg is about names that are incomplete.